src/osd/PGLog.cc

```diff
@@ -996,7 +996,12 @@ void PGLog::rebuild_missing_set_with_deletes(
     dout(20) << __func__ << " check for log entry: " << *i << " = " << r << dendl;
 
     if (r >= 0) {
+#ifndef WITH_SEASTAR
       object_info_t oi(bv);
+#else
+      object_info_t oi;
+      oi.decode_without_oid(bv);
+#endif
```

Comment:
This pathway can't happen for crimson at all at the moment.

Reply:
dropped